content/rest/guides/basics-of-authentication.md

```diff
@@ -76,7 +76,7 @@ Next, in _views/index.erb_, paste this content:
     </p>
     <p>
       We're going to now talk to the GitHub API. Ready?
-      <a href="https://github.com/login/oauth/authorize?scope=user:email&client_id=<%= client_id %>">Click here</a> to begin!</a>
```

Comment:
content/rest/guides/basics-of-authentication.md

Reply:
???

Reply:
???